SingularityUI/app/actions/api/history.es6

```diff
@@ -3,8 +3,9 @@ import Utils from '../../utils';
 
 export const FetchTaskHistory = buildApiAction(
   'FETCH_TASK_HISTORY',
-  (taskId) => ({
-    url: `/history/task/${taskId}`
+  (taskId, isMainApiCall) => ({
+    url: `/history/task/${taskId}`,
+    isMainApiCall
```

Comment:
`isMainApiCall` doesn't convey that information at all -- if that's all that this variable controls, can we think of a more descriptive name for it?